cwms-data-api/src/main/java/cwms/cda/data/dto/TimeSeries.java

```diff
@@ -118,19 +121,20 @@ public class TimeSeries extends CwmsDTOPaginated {
     private TimeSeries() {}
 
     public TimeSeries(String page, int pageSize, Integer total, String name, String officeId, ZonedDateTime begin, ZonedDateTime end, String units, Duration interval) {
-        this(page, pageSize, total, name, officeId, begin, end, units, interval, null, null, null);
+        this(page, pageSize, total, name, officeId, begin, end, null, units, interval, null, null, null);
     }
 
-    public TimeSeries(String page, int pageSize, Integer total, String name, String officeId, ZonedDateTime begin, ZonedDateTime end, String units, Duration interval, VerticalDatumInfo info){
-        this(page, pageSize, total, name, officeId, begin, end, units, interval, info, null, null);
+    public TimeSeries(String page, int pageSize, Integer total, String name, String officeId, ZonedDateTime begin, ZonedDateTime end, String units, Duration interval, VerticalDatumInfo info, Timestamp versionDate){
+        this(page, pageSize, total, name, officeId, begin, end, versionDate, units, interval, info, null, null);
     }
 
-    public TimeSeries(String page, int pageSize, Integer total, String name, String officeId, ZonedDateTime begin, ZonedDateTime end, String units, Duration interval, VerticalDatumInfo info, Long intervalOffset, String timeZone) {
+    public TimeSeries(String page, int pageSize, Integer total, String name, String officeId, ZonedDateTime begin, ZonedDateTime end, Timestamp versionDate, String units, Duration interval, VerticalDatumInfo info, Long intervalOffset, String timeZone) {
```

Comment:
If begin and end are ZonedDateTime I'd keep versionDate as ZonedDateTime too.

Reply:
Done